test/e2e/cypress/e2e/host_details.cy.js

```diff
@@ -355,4 +355,41 @@ context('Host Details', () => {
       cy.get('span').find('svg').should('exist');
     });
   });
+
+  describe('Deregistration', () => {
+    describe('"Clean up" button should be visible only for an unhealthy host', () => {
+      it('should not display the "Clean up" button for healthy host', () => {
+        cy.task('startAgentHeartbeat', [selectedHost.id]);
+        cy.contains('button', 'Clean up').should('not.exist');
```

Comment:
Nice! Actually I prefer this method over the one @rtorrero implemented in the host list view hehe
But to keep the same, we should still use other way around. Ruben added a `data-testid` so, let's use it I guess.
What do you think @rtorrero ?